instructor/common.py

```diff
@@ -274,7 +282,9 @@ def assignments_variables(request):
     form = AssignmentForm(instance=assignment)
     return render_to_response('instructor/assignment_select_variables.html',
                               {'form': form,
-                               'assignment': assignment
+                               'treatments_created': treatments_created,
```

Comment:
It may be cleaner to pass this as a JSON string so it can be directly included in the template. Can you use `json.dumps` on `treatments_created` here? Then see comment below